i18n/ja/docusaurus-plugin-content-docs/current/guides/integration-mobile-client-server.md

```diff
@@ -0,0 +1,361 @@
+
+- [UID2 SDK for Android Reference Guide](../sdks/uid2-sdk-ref-android.md)
+- [UID2 SDK for iOS Reference Guide](../sdks/uid2-sdk-ref-ios.md)
+
+For example, for Android, follow the UID2 SDK documentation to:
+- Add the pre-built UID2 SDK for Android libraries into your mobile app: see [Installation](../sdks/uid2-sdk-ref-android.md#installation).
+- Learn how to instantiate the UID2Manager instance in your Android app: see [Android Initialization](../sdks/uid2-sdk-ref-android.md#android-initialization).
+ here.
+ 
+ For iOS, refer to the [Usage Guidelines](../sdks/uid2-sdk-ref-ios.md#usage-guidelines).
+
+## Using the UID2 Integration Environment
+
+By default, the SDK is configured to work with the UID2 production environment: `https://prod.uidapi.com`. If you want to use the UID2 integration environment instead, provide the following URL in your call to UID2Manager initialization:
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+```js
+UID2Manager.init(
+  context = this,
+  serverUrl = "https://operator-integ.uidapi.com"
+)
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+// Must be set before UID2Manager.shared is accessed UID2Settings.shared.environment = .custom(
+  url: URL(string: "https://operator-integ.uidapi.com")!
+)
+```
+
+</TabItem>
+</Tabs>
+
+On iOS, you must set `UID2Settings` before you first access `UID2Manager.shared`. Any changes made to settings after the first access are  not read.
+
+:::note
+Bear in mind the following differences between environments:
+- Tokens from the UID2 integration environment are not valid for passing to the bid stream.
+- You'll have a different set of Subscription ID and public key values for each environment (integ and prod). Be sure to use the correct values for each environment.
+:::
+
+## Optional: Reduce Latency by Setting the API Base URL for the Production Environment
+
+<ReduceLatency />
+
+To specify a different UID2 server, you can change it in the `init` call:
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+```js
+UID2Manager.init(
+  context = this,
+  serverUrl = " https://global.prod.uidapi.com"
+)
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+UID2Settings.shared.environment = .singapore
+// or
+UID2Settings.shared.environment = .custom(
+  url: URL(string: "https://global.prod.uidapi.com")!
+)
+```
+
+</TabItem>
+</Tabs>
+
+## Configure the UID2 Mobile SDK for Your Mobile App
+
+After you've instantiated `UID2Manager` correctly in your mobile app, you'll need to pass a UID2 <Link href="../ref-info/glossary-uid#gl-identity">identity</Link> generated server-side (see [Implement server-side token generation](#implement-server-side-token-generation)), and then pass it into the mobile app using the `setIdentity` method, as shown in the following:
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+```js
+UID2Manager.getInstance().generateIdentity()
```

Comment:
should be `setIdentity()` instead

Reply:
@sunnywu not sure of this. I replaced it but it's only part of a line of code. Added a query. Please verify. I'm sorry you reviewed these files twice. Next time I will not copy the files into the Japanese. But then there will be build errors.